src/__tests__/node.test.ts

```diff
@@ -69,5 +69,31 @@ describe('node.ts', () => {
       expect(node.perm).toBe(0o600);
       expect(node.isFile()).toBe(true);
     });
+    describe.each([['uid'], ['gid'], ['atime'], ['mtime'], ['perm'], ['nlink']])('ctime changes', field => {
```

Comment:
You don't need these nested arrays:

```suggestion
    describe.each(['uid', 'gid', 'atime', 'mtime', 'perm', 'nlink'])('ctime changes', field => {
```